pybamm/step/steps.py

```diff
@@ -121,6 +121,7 @@ def current(value, **kwargs):
     ----------
     value : float
         The current value in A.
+        Value can be a number, a 2-tuple (for cccv_ode), or a 2-column array (for drive cycles)
```

Comment:
```suggestion
        The current value in A. It can be a number or a 2-column array (for drive cycles)
```

The cccv_ode is only possible at the `_Step` level so does not need to be included here.

This applies to all below.